microcontroller/Core/Inc/calc.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * calc.h
+ *
+ *  Created on: Apr 21, 2023
+ *      Author: ProfessorG
+ */
+
+#ifndef INC_CALC_H_
+#define INC_CALC_H_
+
+typedef struct {
+	uint16_t steering_wheel_angle;
+	uint16_t throttle_percent;
+	uint16_t velocity_left;
+	uint16_t velocity_right;
+	uint16_t timestamp; // TODO: Format
+} sensors_raw_t;
+
+typedef struct { // TODO: Units
+	float steering_wheel_angle; // degrees
+	float throttle_percent; // ???
+	float velocity_left; // ???
```

Comment:
And velocity can be in MPH once converted from frequency.
Also, we are going to capture frequency from the front wheels. (velocity_fr, velocity_fl).
And, we are going to get speed from the rear wheels through CAN communication. (velocity_rr, velocity_rl).